client/components/mma/switch/SwitchComplete.tsx

```diff
@@ -74,7 +75,19 @@ export const SwitchComplete = () => {
 			</section>
 			{!switchContext.isFromApp && (
 				<section css={sectionSpacing}>
-					<SignInBanner />
+					<Stack space={6}>
+						<SignInBanner />
+						<div>
+							<LinkButton
+								cssOverrides={css`
+									margin-top: ${space[6]}px;
+								`}
+								href="https://www.theguardian.com/"
+							>
```

Comment:
Sorry, might have confused things with my last comment! 😅 The `<Stack>` is applying spacing between the banner and button now so you can remove the override for `margin-top` completely.

```suggestion
							<LinkButton href="https://www.theguardian.com/">
```

Reply:
oh right I thought we needed it as you suggested changing the css to cssOverride, but that make sense if Stack is doing the work